include/ofi_hmem.h

```diff
@@ -62,6 +62,8 @@ CUresult ofi_cuGetErrorName(CUresult error, const char** pStr);
 CUresult ofi_cuGetErrorString(CUresult error, const char** pStr);
 CUresult ofi_cuPointerGetAttribute(void *data, CUpointer_attribute attribute,
 				   CUdeviceptr ptr);
+CUresult ofi_cuDeviceGetP2PAttribute(int* value, CUdevice_P2PAttribute attrib,
```

Comment:
For consistency with the rest of the code - `int *value`

Reply:
I'll change this one, but not all the type signatures here "bind" `*`/`**` to the variable (probably just pasted from the NVIDIA docs like I did here). I'll go ahead and change those as well.

Reply:
I opted not to change all the other type signatures since I like to preserve the line-by-line Git blame as much as possible.